src/Sarif.Sarifer/SpamBackgroundAnalyzer.cs

```diff
@@ -38,6 +38,11 @@ public SpamBackgroundAnalyzer()
 
         protected override bool AnalyzeCore(Uri uri, string text, string solutionDirectory, SarifLogger sarifLogger, CancellationToken cancellationToken)
         {
+            if (!SariferOption.Instance.IfAnalyzeSarifFile && uri.GetFilePath().EndsWith(".sarif"))
```

Comment:
>EndsWith(".sarif")) [](start = 80, length = 19)

add ignore case: StringComparison.OrdinalIgnoreCase

Reply:
good catch, fixed

---
In reply to: [564968969](https://github.com/microsoft/sarif-visualstudio-extension/pull/354#discussion_r564968969) [](ancestors = 564968969)